README.md

```diff
@@ -51,6 +51,86 @@ $ rails db:migrate
  $ rails s
  ```
 
+## MySQL permissions setup
+
+You may see an error about MySQL permissions. If you do, you need to set up your user's permissions within MySQL.
+
+You'll want to log in to MySQL as `root` so you can make these changes. With MySQL running, do:
+
+```bash
+$ sudo mysql
+```
+Enter your password (the one you normally use with `sudo`, not a MySQL related password). You'll be in the MySQL console.
+
+Change to the system database for MySQL.
+
+```sql
+mysql> use mysql;
```

Comment:
I don't think this step is really necessary. Is it?

Reply:
It was for me, I got a "no database specified" error otherwise. Try it out on your machine -- I can make a note that it's optional if you don't need to do it.

Reply:
Oh, yes, I see. I just like to do the whole thing as `root`, which made life easier -- no grants needed.